docs/content/docs/user-docs/openshift.md

```diff
@@ -109,7 +109,22 @@ Delete `config.txt` and `secrets.txt`.
 If you change the name of either the `ConfigMap` or the `Secret` from the values given above, i.e. `ack-$SERVICE-user-config` and `ack-$SERVICE-user-secrets`, then installations from OperatorHub will not function properly. The Deployment for the controller is preconfigured for these key values.
 {{% /hint %}}
 
-### Step 4: Install the controller
+### Step 4 (Optional): Apply Additional Custom Resource Definitions(CRD)
+CRDs are global resources, they cannot be shared, or installed by multiple controllers. This is why [AdoptedResource](../../../reference/common/v1alpha1/adoptedresource/) and [FieldExport](../../../reference/common/v1alpha1/fieldexport/)
+are not packaged with any of the ACK controllers within OpenShift. If either of these resources is required, they will need to be applied manually to the cluster
+before the operator is installed.
+
+Apply the `AdoptedResoutce` CRD
+```bash
+oc apply -f https://raw.githubusercontent.com/aws-controllers-k8s/runtime/main/config/crd/bases/services.k8s.aws_adoptedresources.yaml
```

Comment:
I'm not familiar with these resources off hand, aside from the GVK versioning, is there a reason why we should not point to `main`'s representation of the AdoptedResource or FieldExport YAMLs? As in, is there any reason why we would need to point to a different tag/branch for this manifest?

Reply:
It is very unlikely that `AdoptedResource` and `FieldExport` will receive some updates, at least in the short to mid-term. I would say that it's completely fine to point to `main` for now. 

The best way to prevent any issues is by pointing to the runtime version used in a controller's `go.mod`.

Reply:
I think for this documentation `main` is really the only thing we have, and if new fields are added, the controller will still use the `old` CRD and be able to reconcile the resource, until the GVK changes. This really isn't the ideal way to document this, ideally it would be tied to version, but I think it's the best we have.